hubblestack/files/hubblestack_nova/win_firewall.py

```diff
@@ -154,7 +154,7 @@ def _get_tags(data):
 def _export_firewall():
     dump = []
     try:
-        temp = __salt__['cmd.run']('Get-NetFirewallProfile -PolicyStore ActiveStore', shell='powershell', python_shell=True)
+        temp = __salt__['cmd.run']('mode con:cols=1000 lines=1000; Get-NetFirewallProfile -PolicyStore ActiveStore', shell='powershell', python_shell=True)
```

Comment:
following statement 
```temp = __salt__['cmd.run']('Get-NetFirewallProfile -PolicyStore ActiveStore', shell='powershell', python_shell=True)```
gives the output like
```
NotifyOnListen                  : False
EnableStealthModeForIPsec       : True
LogFileName                     : %systemroot%\system32\logfiles\firewall\domai
                                  nfw.log
LogMaxSizeKilobytes             : 16384
LogAllowed                      : True
LogBlocked                      : True
```

notice that the value of LogFileName is printed in two lines.  This causes failure in windows audit. 
```
 'Failure': [{'CIS-9.3.7': {'description': "(l1) ensure 'windows firewall - public - logging -  name' is set to '%systemroot%\\system32\\logfiles\\firewall\\publicfw.log'",
                            'found_value': '%systemroot%\\system32\\logfiles\\firewall\\publi',
                            'match_output': '%SYSTEMROOT%\\System32\\logfiles\\firewall\\publicfw.log',
                            'module': 'win_auditpol',
                            'name': 'LogFileName',
                            'nova_profile': 'windows-2012r2-level-1-scored-v2-2-1',
                            'tag': 'CIS-9.3.7',
                            'type': 'whitelist',
                            'value_type': 'public'}},
```